src/main/antlr4/org/reaktivity/nukleus/maven/plugin/internal/parser/Nukleus.g4

```diff
@@ -161,6 +169,8 @@ member_list
 
 member
    : type_spec declarators SEMICOLON
+   | unsigned_integer_type declarator EQUALS uint_literal SEMICOLON
+   | signed_integer_type declarator EQUALS int_literal SEMICOLON
```

Comment:
Suggest breaking these into `uint_member` and `int_member` rules.

Reply:
done